test/ex_unit_properties_test.exs

```diff
@@ -72,6 +74,52 @@ defmodule ExUnitPropertiesTest do
       assert Agent.get(counter, & &1) == 10
     end
 
+    property "runs for the specified number of milliseconds" do
+      {:ok, counter} = Agent.start_link(fn -> 0 end)
+
+      # I'm explicitly passing max_runs: :not_set to override the project-wide
+      # config in the `mix.exs` file of `max_runs: 100`.
+      check all :ok <- :ok, max_runs: :not_set, max_run_time: 250 do
+        :timer.sleep(1)
+        Agent.update(counter, &(&1 + 1))
```

Comment:
What if you use `max_run_time: 250` and use `Process.sleep(100)` inside the block, and then assert that we did exactly two runs? It should be much more precise when using bigger intervals than something like 1ms.

Reply:
I wanted a case with more than 100 runs to make sure that we're not accidentally cutting off at 100 runs even if we shouldn't be (since I almost accidentally did that while implementing this feature). It just made me feel safe, but maybe it's overkill? If you think it's overkill then I think we can get rid of this entire test and just rely on the other test since that covers things pretty well.